README.md

```diff
@@ -1,2 +1,35 @@
 # Hello-Microverse
-Set up Hello Microverse' HTML project
+Hello-Microverse
+First Microverse project.
+
+Built With
+HTML and CSS
+
+
+Getting Started
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+
+👤 Author
+GitHub: @tarak-psdu
+Twitter: @AbuTarak10
+LinkedIn: linkedin.com/in/abu-tarak
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+@Vgonma
+@ceddddy
+
+📝 License
```

Comment:
- [ ] Please you should use hashtags for the titles in your `readme.md` file to make it more readable instead of using raw text. :pray: